src/tripleanddouble.js

```diff
@@ -0,0 +1,35 @@
+// the function for the extention
+
+// triple word = {dog}    = 15 points
+// triple letter =d{o}g   = 7 points
+
+// double word = [dog]    = 10 points
+// double letter =d[o]g   = 6 points
+
+// function tripleOrDoubleWord (word){
+//     //     if (word[0] === '{' && word[word.length - 1] === '}') {
+//     //         return scrabble(word)* 2
+//     //     } else if (word[0] === '[' && word[word.length - 1] === ']') {
+//     //         return scrabble(word)* 3
+//     // } else {
+//     //     return scrabble(word)
+//     // 
+// }
+// }
+
+
+// function doubleWordScore(word) {
```

Comment:
hello. Not sure why you don't consider this as a running code, in my opinion double and triple score for a whole word is correct..

Reply:
I tried to run both of the codes, the function with both double and triple and the 2 when they are separated, but they would constantly come up with an error. So I will try again today